deploy/001-source-timelocks.ts

```diff
@@ -18,60 +19,32 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     return live ? governorBravoDelegatorAddress : deployer;
```

Comment:
Line 16 fails on hardhat

Reply:
I wrote scripts for this here - https://github.com/VenusProtocol/subgraphs/blob/develop/deploy/018-governance.ts

Reply:
cb409e8a0fb030b26441249c98c7f018e9a13c3a